<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateProject</class>
 <widget class="QWidget" name="DialogCreateProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="formWidget" native="true">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="projectNameLabel">
           <property name="text">
            <string>Project Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="projectNameLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="projectModIDLabel">
           <property name="text">
            <string>Mod ID</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="projectModIDLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="projectAuthorLabel">
           <property name="text">
            <string>Project Author</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="projectAuthorLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="projectDescriptionLabel">
           <property name="text">
            <string>Project Description</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="projectDescriptionLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="projectVersionLabel">
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="projectVersionLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="dialogButtons">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectNameLineEdit</tabstop>
  <tabstop>projectDescriptionLineEdit</tabstop>
  <tabstop>projectModIDLineEdit</tabstop>
  <tabstop>projectAuthorLineEdit</tabstop>
  <tabstop>projectVersionLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
